pkg/utils/codegeneration/utils.go

```diff
@@ -0,0 +1,39 @@
+package codegeneration
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"strings"
+
+	"golang.org/x/tools/imports"
+)
+
+func PrintFiles(dir string, files map[string]string) error {
```

Comment:
How about `Write` instead of `Print`?
```suggestion
// WriteFiles writes the map of files to dir and runs goimports on all *.go files.
func WriteFiles(dir string, files map[string]string) error {
```